editor/animation_track_editor.cpp

```diff
@@ -1239,6 +1239,59 @@ void AnimationMultiTrackKeyEdit::set_use_fps(bool p_enable) {
 }
 
 void AnimationTimelineEdit::_zoom_changed(double) {
```

Comment:
Shouldn't the variables here be `float`? Most methods called here use float.

Reply:
I am not sure. `get_value()` returns a double, and `hscroll` (derives `Range`) expects a double, so I thought I'd just do all calculations with doubles. Should I change everything to be a float instead?